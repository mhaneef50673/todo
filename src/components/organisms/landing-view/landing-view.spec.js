import React from 'react';
import { mount } from 'enzyme';
import { mountToJson} from 'enzyme-to-json';
import LoaderView from './index';

describe('LoaderView Component', () => {
  it('should render the component', () => {
    const tree = mount(
      <LoaderView />
    )
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});