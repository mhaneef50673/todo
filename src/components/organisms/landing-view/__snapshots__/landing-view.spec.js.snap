// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderView Component should render the component 1`] = `
<LandingView>
  <div
    className="landing-container"
  >
    <div
      className="loader-container"
    >
      <LoaderView>
        <div>
          <div
            className="app-logo-container"
          >
            <AnimatedImage
              extraClass="app-logo"
              imageUrl="app_logo.png"
              shouldUseRippleEffect={true}
              showRippleEffect={false}
            >
              <div
                className="image-container"
              >
                <img
                  alt="animated"
                  className="app-logo"
                  src="app_logo.png"
                />
                <span
                  className="ripple false"
                />
              </div>
            </AnimatedImage>
          </div>
          <div
            className="progress-bar-container"
          >
            <ProgressBar
              percentage={0}
            >
              <div
                className="progress-bar"
              >
                <div
                  className="filler"
                  style={
                    Object {
                      "width": "0%",
                    }
                  }
                />
              </div>
            </ProgressBar>
          </div>
        </div>
      </LoaderView>
    </div>
  </div>
</LandingView>
`;
